FlowCrypt/App/PromiseExtensions.swift

```diff
@@ -24,6 +25,16 @@ extension Promise {
     }
 }
 
+extension Future {
+    static func resolveAfter<T, E: Error>(timeout: TimeInterval = 5, with result: Result<T, E>) -> Future<T, E> {
```

Comment:
This looks like it belongs in test folder?

Reply:
good point. Will move to test target